.vale.ini

```diff
@@ -0,0 +1,11 @@
+StylesPath = .vale/styles
+
+Vocab = Silverstripe
+
+MinAlertLevel = suggestion
+
+; Don't check inside heading anchors, i.e. inside {#my-anchor}
+WordTemplate = "[^#]\b(?:%s)\b[^}]"
+
+[*.md]
+BasedOnStyles = Silverstripe-cms, openly #Vale
```

Comment:
Add blank line at end of file

Reply:
Marking resolved since this is now out of scope.